### PR TITLE
Update coding standards, run phpcbf

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

File: inc/console/abstractcommand.class.php [hunk 1]
```diff
@@ -90,7 +89,7 @@ protected function initDbConnection() {
       global $DB;
 
       if ($this->requires_db && (!($DB instanceof DB) || !$DB->connected)) {
-         throw new RuntimeException(__('Unable to connect to database.'));
+         throw new \Symfony\Component\Console\Exception\RuntimeException(__('Unable to connect to database.'));
```
Comment (trasher) [inline inc/console/abstractcommand.class.php:92]: This one sounds strange :D

File: inc/config.class.php [hunk 2]
```diff
@@ -1558,7 +1558,7 @@ static function validatePassword($password, $display = true) {
       global $CFG_GLPI;
 
       $ok = true;
-      $exception = new PasswordTooWeakException();
+      $exception = new \Glpi\Exception\PasswordTooWeakException();
```
Comment (trasher) [inline inc/config.class.php:1561]: Should use either the import or the full name, not both

**Comment by cedric-anne** [comment]
Was already done in #6830.

File: inc/console/abstractcommand.class.php (hunk 1 above)
Comment (cedric-anne) [reply inc/console/abstractcommand.class.php:92]: This is why this rule exists. Many libs uses same names for exceptions (like `RuntimeException` or `NotFoundException`). Forcing usage of full path will prevent ambiguity/mistakes.

**Pull request closed without merging**